werkzeug/formparser.py

```diff
@@ -11,7 +11,14 @@
 """
 import re
 import codecs
-from tempfile import SpooledTemporaryFile
+try:
+    from tempfile import SpooledTemporaryFile
```

Comment:
just a note here, running GAE locally, this line does not trigger `ImportError`, although trying to use the module does cause an error (so, essentially, this workaround doesn't work locally. i'm not sure if it worked on GAE infrastructure.)

Reply:
but—huge thanks to whoever did this work to ensure compatibility with GAE. you're a lifesaver.